azurelinuxagent/common/cgroupapi.py

```diff
@@ -67,9 +67,9 @@ def track_cgroups(extension_cgroups):
                         "Error: {1}".format(cgroup.path, ustr(exception)))
 
     @staticmethod
-    def _get_extension_cgroup_name(extension_name):
+    def get_extension_cgroup_name(extension_name):
         # Since '-' is used as a separator in systemd unit names, we replace it with '_' to prevent side-effects.
-        return extension_name.replace('-', '_')
+        return "azure-vmextensions-" + extension_name.replace('-', '_')
```

Comment:
let's use CGroupConfigurator._VMEXTENSIONS_SLICE instead

Reply:
It has `.slice` suffix which we don't want.

Reply:
let's declare another constant in the same place (minus ".slice") or remove ".slice" here

Reply:
This is creating cycling dependency issue.. I have created constant in the cgroupapi Module.